### PR TITLE
re-org dirs

### PR DESCRIPTION
**Pull request by duglin**
I also renamed the "spec.md" file to be "cloudevents.md"

Also slipped in a fix for #871

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,40 +27,41 @@ The following documents are available:
 |                               |                                 Latest Release                                  |                                      Working Draft                                       |
 | :---------------------------- | :-----------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------: |
 | **Core Specification:**       |
-| CloudEvents                   |          [v1.0.1](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md)          |            [master](https://github.com/cloudevents/spec/blob/master/spec.md)             |
+| CloudEvents                   |          [v1.0.1](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md)          |            [WIP](https://github.com/cloudevents/spec/blob/master/cloudevents/cloudevents.md)             |
```
[the remark this answers is not in the document]
Comment (duglin) [reply README.md:30]: it does?

**Comment by duglin** [comment]
I went back and forth on the spec.md rename and decided to pull the trigger because (while I agree that the dir name provides context), there are times when I've run into situations where people end up having these "commonly named things" next to each other - and then conflicts arise. For example, they download both files independently w/o the entire dir structure.  But let's discuss on the call today

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jskeet) [reply README.md:30]: Nope, I'd misread. Ignore me.

**Comment by duglin** [comment]
use common names - e.g. spec.md for all specs

**Comment by duglin** [comment]
Approved on the Nov 4th call with the xxx.md -> spec.md change

**Pull request merged**